validation-test/Sema/type_checker_perf/fast/rdar46541800b.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -solver-enable-operator-designated-types
+
+import SIMDOperators
```

Comment:
Thanks for adding the test-case! I think this should be `simd` since `SIMDOperators` got merged back?

Reply:
We shouldn't need any import at all with the operators moved back into the stdlib.